tests/Avalonia.Controls.UnitTests/CarouselTests.cs

```diff
@@ -16,12 +20,7 @@ public void First_Item_Should_Be_Selected_By_Default()
         {
             var target = new Carousel
             {
-                Template = new FuncControlTemplate<Carousel>(CreateTemplate),
-                Items = new[]
-                {
-                    "Foo",
-                    "Bar"
-                }
+                Template = new FuncControlTemplate<Carousel>(CreateTemplate), Items = new[] { "Foo", "Bar" }
```

Comment:
Can you revert all of these formatting changes combining everything into one line? Here and below. There is also one place in SelectingItemsControl.

Reply:
I will fix it.